src/main/java/org/cryptomator/cryptofs/dir/C9rConflictResolver.java

```diff
@@ -111,6 +121,7 @@ private Node renameConflictingFile(Path canonicalPath, Path conflictingPath, Str
 		Node node = new Node(alternativePath);
 		node.cleartextName = alternativeCleartext;
 		node.extractedCiphertext = alternativeCiphertext;
+		eventConsumer.accept(new ConflictResolvedEvent(cleartextPath.resolve(cleartext), canonicalPath, cleartextPath.resolve(alternativeCleartext), alternativePath));
```

Comment:
third param `cleartextPath.resolve(alternativeCleartext)` is either not what the event object wants here or all the different properties are pretty confusing.

> oldVersionCleartextPath path within the cryptographic filesystem of the renamed resource

Is there really such a thing as an "old cleartext"? And is there any benefit of displaying this to the user? I'd drop this.

Reply:
old is from the point of syncing 😅 I'll rename the parameters

The event should give the library consumer as much information as possible. Therefore, i added 4 parameters:
* the old, conflicting ciphertext name
* the cleartext name of the conflicting resource
* the new, non-conlict ciphertext name
* the new cleartext name

Especially with our new rename algorithm in #275 we should keep the cleartextname of the the conflicting resource.

Reply:
Changed in https://github.com/cryptomator/cryptofs/pull/277/commits/2cd07fae1fc779bfa798ba287919ba9cb69227af.